HedgeModManager/Languages/pt-PT.xaml

```diff
@@ -107,13 +107,13 @@
     <system:String x:Key="StatusUIDeletedMod"             >{0} apagado</system:String>
     <system:String x:Key="StatusUIModsDBSaved"            >Base de dados de mods guardada</system:String>
     <system:String x:Key="StatusUIModsDBLocationChanged"  >Localização da base de dados atualizada</system:String>
-    <system:String x:Key="StatusUIGameChange"             >Jogo mudado para {0}</system:String>
+    <system:String x:Key="StatusUIGameChange"             >Jogo alterado para {0}</system:String>
     <system:String x:Key="StatusUIDownloadingCodes"       >A transferir códigos para {0}</system:String>
     <system:String x:Key="StatusUIDownloadFinished"       >Transferência terminada</system:String>
     <system:String x:Key="StatusUIDownloadFailed"         >A transferência falhou</system:String>
 
     <!-- Dialog UI Strings -->
-    <system:String xml:space="preserve" x:Key="DialogUIDeleteMod">Tens a certeza de que pretendes apagar "{0}"?&#x0a;Esta ação não pode ser desfeita!</system:String>
```

Comment:
I feel it's better to keep "desfeita" instead of changing to "anulada" in this context...

Reply:
Both are ok but anulada is used more in portuguese and because of that i feel it's more natural. Sinceramente seja qual for o resultado final fico contente na mesma.

Reply:
![Capturar](https://user-images.githubusercontent.com/18236868/83255871-c464c880-a1a0-11ea-9f1d-5074b2aaf161.PNG)
Por exemplo